### ISSUE TITLE
[Release][Tune] Migrate `tune_scalability_result_throughput_single_node` tests to v2 stack and Anyscale Jobs

### ISSUE THREAD

**Issue by Yard1** [ray-project/ray#32716]
### What happened + What you expected to happen

Migrate `tune_scalability_result_throughput_single_node` test to v2 stack and Anyscale Jobs. If migration is impossible, consider rewriting the test or removing it. If there are features missing in v2 stack or/and Anyscale Jobs that make the migration impossible, please outline them in this issue.

### Versions / Dependencies

master

### Reproduction script

-

### Issue Severity

None

**Comment by shomilj** [comment]
Copying the failure here ~ we are seeing higher overhead on V2, which is what is causing this to fail:
```
[INFO 2023-03-06 14:13:15,463] log.py: 41  Observed the following results:
 
  time_taken = 121.367389457
  trial_states = {'TERMINATED': 96}
  last_update = 1678140756.8763
  last_update_diff = 7.180083751678467
 
[INFO 2023-03-06 14:13:15,463] artifacts.py: 38  Wrote test config and result to artifacts directory: /tmp/artifacts
[ERROR 2023-03-06 14:13:15,464] run_release_test.py: 164  Took too long to complete (time_taken=121.37 > 120).
```

**Comment by shomilj** [comment]
Can we just bump the 120s to 122s to account for Jobs overhead? cc @krfricke

**Comment by krfricke** [comment]
Closed by #33873 - thanks @can-anyscale!
